src/components/Sketches/components/CreateSketchModal.js

```diff
@@ -43,6 +46,18 @@ class CreateSketchModal extends React.Component {
     });
   };
 
+  changeLanguage = (lang) => {
+    this.setState({ language: lang });
+  };
+
+  dropDownValues = LanguageDropdownValues.map(({ display, value }) => {
```

Comment:
Sorry for being unclear earlier. Don’t like making you undo work, but think about it this way. In the future if you or someone else wants to include a dropdown button in a page, it wouldn’t make sense for them to essentially write the rendering logic again.